From ed31960498c8ab85c558b3d47f164becd4e01b35 Mon Sep 17 00:00:00 2001
From: Matt Jolly <Matt.Jolly@footclan.ninja>
Date: Thu, 26 Oct 2023 00:57:43 +1000
Subject: [PATCH 2/2] Move state info to run subdirectory

---
 src/library/paths.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/library/paths.h b/src/library/paths.h
index f066f38..63427ce 100644
--- a/src/library/paths.h
+++ b/src/library/paths.h
@@ -34,7 +34,7 @@
 #define DB_NAME         "trust.db"
 #define REPORT          "/var/log/fapolicyd-access.log"
 #define RUN_DIR         "/run/fapolicyd/"
-#define STAT_REPORT     "/run/fapolicyd.state"
+#define STAT_REPORT     "/run/fapolicyd/fapolicyd.state"
 #define fifo_path       "/run/fapolicyd/fapolicyd.fifo"
 #define pidfile         "/run/fapolicyd.pid"
 
-- 
2.42.0

